build.gradle

```diff
@@ -35,15 +36,22 @@ application {
 
 // ### Checkstyle plugin settings
 checkstyle {
-    toolVersion = "8.36.2"
+    toolVersion = '10.2'
```

Comment:
Big thanks for updating all these, and the `google_checks` too!

Reply:
You're welcome!